modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -6262,6 +6262,7 @@ document-types=Document Types
 document-updated-email=Document Updated Email
 document-util=Document Util
 document-x-of-x=Document {0} of {1}
+documentation=documentation
```

Comment:
@gislayne-vitorino doesn't the value need to be capitalized?

Reply:
@david-truong Yeah, that's what I thought, but I wasn't sure how I could do it differently and still match the design